src/rules/percent-placeholder-pattern/index.js

```diff
@@ -0,0 +1,70 @@
+import _ from "lodash"
```

Comment:
Should we just import the functions that we need from lodash, like we do in other rules?

```js
import { isRegExp, isString } from "lodash"
```